superset-frontend/plugins/legacy-plugin-chart-world-map/src/WorldMap.js

```diff
@@ -165,12 +168,14 @@ function WorldMap(element, props) {
     }
   };
 
-  const handleContextMenu = source => {
+  const handleContextMenu = async source => {
```

Comment:
Nit in addition to Ville's comments - I think we don't need that `async` here anymore?

Reply:
Same in other charts